controllers/nodefeaturediscovery_resources.go

```diff
@@ -69,34 +82,65 @@ func filePathWalkDir(root string) ([]string, error) {
 	return files, err
 }
 
+// getAssetsFrom takes a path as an input and grabs all of the
+// file names in that path, then returns a list of the manifests
+// it found in that path.
```

Comment:
Again, suggest a bit of streamlining the wording

```suggestion
// getAssetsFrom recursively reads all manifest files under a given path
```